smqtk_relevancy/impls/libsvm_hik.py

```diff
@@ -9,10 +9,11 @@
 
 from smqtk_relevancy.interfaces.relevancy_index import RelevancyIndex
 from smqtk_descriptors import DescriptorElement
-from smqtk_relevancy.utils.distance_kernel import (
+from smqtk_relevancy.utils.distance_kernel import (  # type: ignore
```

Comment:
This ignore should not be needed. You probably need to add a `py.typed` (empty) file to the module: `smqtk_relevancy/py.typed`.

Reply:
See https://mypy.readthedocs.io/en/stable/installed_packages.html#making-pep-561-compatible-packages

Reply:
Poetry conveniently already knows to include this file in build packages, so we don't have to edit anything like their documented setup.py update.